notebooks/Understand_Tables.ipynb

```diff
@@ -0,0 +1,4387 @@
+{
```

Comment:
<p>wrap as a series to display better?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>Replaced with <code>drop_duplicates()</code>  on a one-column DataFrame. Displays better now.</p>